src/collectors/windows.plugin/perflib-netframework.c

```diff
@@ -0,0 +1,878 @@
+
+        if (perflibGetObjectCounter(pDataBlock, pObjectType, &p->NETFrameworkCLRInteropCOMCallableWrappers)) {
+            snprintfz(id, RRD_ID_LENGTH_MAX, "%s_clrinterop_com_callable_wrappers", windows_shared_buffer);
+            if (!p->st_clrinterop_com_callable_wrappers) {
+                p->st_clrinterop_com_callable_wrappers = rrdset_create_localhost("netframework"
+                                                                                 , id, NULL
+                                                                                 , "interop"
+                                                                                 , "netframework.clrinterop_com_callable_wrappers"
+                                                                                 , "COM callable wrappers (CCW)"
+                                                                                 , "ccw/s"
+                                                                                 , PLUGIN_WINDOWS_NAME
+                                                                                 , "PerflibNetFramework"
+                                                                                 , PRIO_NETFRAMEWORK_CLR_INTEROP_CCW
+                                                                                 , update_every
+                                                                                 , RRDSET_TYPE_LINE
+                                                                                 );
+
+                snprintfz(id, RRD_ID_LENGTH_MAX, "netframework_%s_clrinterop_com_callable_wrappers_total", windows_shared_buffer);
+                p->rd_clrinterop_com_callable_wrappers  = rrddim_add(p->st_clrinterop_com_callable_wrappers,
+                                                                    id,
+                                                                    "com_callable_wrappers",
+                                                                    1,
+                                                                    1,
+                                                                    RRD_ALGORITHM_INCREMENTAL);
+
+                rrdlabels_add(p->st_clrinterop_com_callable_wrappers->rrdlabels,
+                              "process",
+                              windows_shared_buffer,
+                              RRDLABEL_SRC_AUTO);
+            }
+
+            rrddim_set_by_pointer(p->st_clrinterop_com_callable_wrappers,
+                                  p->rd_clrinterop_com_callable_wrappers,
+                                  (collected_number)p->NETFrameworkCLRInteropCOMCallableWrappers.current.Data);
+            rrdset_done(p->st_clrinterop_com_callable_wrappers);
+        }
+
+        if (perflibGetObjectCounter(pDataBlock, pObjectType, &p->NETFrameworkCLRInteropMarshalling)) {
+            snprintfz(id, RRD_ID_LENGTH_MAX, "%s_clrinterop_interop_marshalling", windows_shared_buffer);
```

Comment:
Can be moved after the if -- please check the rest as well

Reply:
Done, thanks!